homeassistant/components/pi_hole/binary_sensor.py

```diff
@@ -42,7 +42,7 @@ class PiHoleBinarySensorEntityDescription(
     PiHoleBinarySensorEntityDescription(
         # Deprecated, scheduled to be removed in 2022.6
```

Comment:
they should be removed in a next PR

Reply:
I would not suggest creating translations in that case, as in, the will just pollute the translation system.

Reply:
sounds reasonable 🙂